yogstation/code/game/gamemodes/gangs/gangs.dm

```diff
@@ -7,8 +7,8 @@ GLOBAL_LIST_EMPTY(gangs)
 	config_tag = "gang"
 	antag_flag = ROLE_GANG
 	restricted_jobs = list("Security Officer", "Warden", "Detective", "AI", "Cyborg","Captain", "Head of Personnel", "Head of Security", "Chief Engineer", "Research Director", "Chief Medical Officer")
-	required_players = 40
-	required_enemies = 1
+	required_players = 1
```

Comment:
debug code moment ;)

could always use the debug - game verb to disable it while testing